CONTRIBUTING.md

```diff
@@ -0,0 +1,167 @@
+# ROCM-CORE Contributing Guide
+To ensure the quality of the ROCM-CORE code base, the ROCM-CORE team has 
+established a code review process to inform developers of the steps 
+that are required to shepherd a change-set into the repository.
+
+#### Table Of Contents
+
+[How to get started](#How-to-get-started)
+
+[How do I contribute?](#how-do-i-contribute)
+  * [Reporting Issues](#reporting-issues)
+  * [Creating a Pull Request](#Creating-a-Pull-Request)
+
+[Responsibility of the Author](#Responsibility-of-the-Author)
+
+[Responsibility of the Reviewer](#Responsibility-of-the-Reviewer)
+
+[The Review](#the-review)
+
+[References](#References)
+## How to get started
+rocm-core is a utility which can be used to get ROCm release version. 
+It also provides the Lmod modules files for the ROCm release.
```

Comment:
Another important reason for it is to provide a base component on which all of rocm can depend, to make it easy to remove all of rocm with a package manager.

Reply:
Updated introduction text  to address the comments in the latest version